libs/positions/src/lib/positions-table.tsx

```diff
@@ -113,234 +124,299 @@ export const AmountCell = ({ valueFormatted }: AmountCellProps) => {
+            if (!value || !data) {
+              return undefined;
+            }
+            if (node?.rowPinned) {
+              return addDecimalsFormatNumber(value, data.decimals);
+            }
+            return data;
+          }}
+        />
+        <AgGridColumn
+          headerName={t('Mark price')}
+          field="markPrice"
+          type="rightAligned"
+          cellRendererSelector={(
+            params: ICellRendererParams
+          ): CellRendererSelectorResult => {
+            return {
+              component: params.node.rowPinned ? EmptyCell : PriceFlashCell,
+            };
+          }}
+          valueFormatter={({
+            value,
+            data,
+            node,
+          }: PositionsTableValueFormatterParams & {
+            value: Position['markPrice'];
+          }) => {
+            if (!data || !value || node?.rowPinned) {
+              return undefined;
+            }
+            if (
+              data.marketTradingMode ===
+              MarketTradingMode.TRADING_MODE_OPENING_AUCTION
+            ) {
+              return '-';
+            }
+            return addDecimalsFormatNumber(
+              value.toString(),
               data.marketDecimalPlaces
-            ),
-            value: range ? Number(((mid - min) * BigInt(100)) / range) : 0,
-            intent: data.lowMarginLevel ? Intent.Warning : undefined,
-          };
-        }}
-      />
-      <AgGridColumn
-        headerName={t('Leverage')}
-        field="currentLeverage"
-        type="rightAligned"
-        cellRenderer="PriceFlashCell"
-        valueFormatter={({
-          value,
-        }: PositionsTableValueFormatterParams & {
-          value: Position['currentLeverage'];
-        }) =>
-          value === undefined ? undefined : formatNumber(value.toString(), 1)
-        }
-      />
-      <AgGridColumn
-        headerName={t('Margin allocated')}
-        field="capitalUtilisation"
-        type="rightAligned"
-        flex={2}
-        cellRenderer="ProgressBarCell"
-        valueFormatter={({
-          data,
-          value,
-        }: PositionsTableValueFormatterParams & {
-          value: Position['capitalUtilisation'];
-        }): PriceCellProps['valueFormatted'] | undefined => {
-          if (!data) {
-            return undefined;
-          }
-          return {
-            low: `${formatNumber(value, 2)}%`,
-            high: addDecimalsFormatNumber(
-              data.totalBalance,
-              data.assetDecimals
-            ),
-            value: Number(value),
-          };
-        }}
-      />
-      <AgGridColumn
-        headerName={t('Realised PNL')}
-        field="realisedPNL"
-        type="rightAligned"
-        cellClassRules={{
-          'text-vega-green-dark dark:text-vega-green': ({
+            );
+          }}
+        />
+        <AgGridColumn
+          headerName={t('Entry price')}
+          field="averageEntryPrice"
+          headerComponentParams={{
```

Comment:
chore (non-blocking): Can we use `headerComponent` here?

Reply:
If we will use header component we will replace default header. We will need to handle sorting and filter menu.